protocols/grpc/health.proto

```diff
@@ -21,6 +21,12 @@ message HealthCheckResponse {
 	ServingStatus status = 1;
 }
 
+message VersionCheckResponse {
+	string grpc_version = 1;
+	string impl_version = 2;
```

Comment:
Could we please rename that one to `agent_version` ?

Reply:
done.